apps/test/unit/templates/instructions/InlineAudioTest.js

```diff
@@ -14,7 +14,16 @@ const DEFAULT_PROPS = {
   style: {
     button: {},
     buttonImg: {}
-  }
+  },
+  ttsAutoplayEnabled: false
+};
+
+// when this function is called and awaited, it will not
+// execute until all promises queued up before it are resolved
```

Comment:
NIT: this grammer is kinda hard to parse

Reply:
Ah yeah it was hard to explain, I'll try again.